com.unity.render-pipelines.high-definition/Runtime/Lighting/LightEvaluation.hlsl

```diff
@@ -421,7 +421,7 @@ SHADOW_TYPE EvaluateShadow_Punctual(LightLoopContext lightLoopContext, PositionI
                                     LightData light, BuiltinData builtinData, float3 N, float3 L, float4 distances)
 {
 #ifndef LIGHT_EVALUATION_NO_SHADOWS
-    SHADOW_TYPE shadow  = 1.0;
```

Comment:
how did this happen?

Reply:
In previous PR I have made the change. It was actually the right thing to do, but after discussing with Francesco, I was willing to come back to a single float even if SHADOW_TYPE  is float3 to save ALU as long as we don't have supported of colored point light shadow.